src/modules/Cluster.rb

```diff
@@ -722,22 +709,12 @@ def Export
       Ops.set(result, "csync2_host", @csync2_host)
       Ops.set(result, "csync2_include", @csync2_include)
       if SCR.Read(path(".target.size"), "/etc/corosync/authkey") != -1
-        out = Convert.to_map(
-          SCR.Execute(
-            path(".target.bash_output"),
-            "uuencode -m /etc/corosync/authkey /dev/stdout"
-          )
-        )
-        Ops.set(result, "corokey", Ops.get_string(out, "stdout", ""))
+        data = File.read("/etc/corosync/authkey")
+        Ops.set(result, "corokey", Base64.encode64(data))
       end
-      if SCR.Read(path(".target.size"), "/etc/csync2/key_hagroup") != -1
-        out = Convert.to_map(
-          SCR.Execute(
-            path(".target.bash_output"),
-            Ops.add(Ops.add("uuencode -m ", @csync2_key_file), " /dev/stdout ")
-          )
-        )
-        Ops.set(result, "csync2key", Ops.get_string(out, "stdout", ""))
+      if SCR.Read(path(".target.size"), @csync2_key_file) != -1
```

Comment:
You can use the native `File.exist?` here:
```suggestion
      if File.exist?(@csync2_key_file)
```